CONTRIBUTING.md

```diff
@@ -137,14 +137,16 @@ Once the initial build on the command line is completed, you should use vscode i
 These two options can be reached in VsCode in the bottom bar (look for triangle), or by typing `control-P Debug <space> ` and then selecting either `Build and Launch` or `Watch`
 
 The new extension window will appear with window title `Extension development host`
+
 ### Working with FSAC
 
 1. Run `build.cmd Build` \ `build.sh Build`
-1. Open Ionide-vscode-fsharp in VSCode.
-2. Set `devMode` to `true` in `src/Core/LanguageService.fs`
-3. Open FSAC in VS
-4. Start FSAC.Suave in VS
-5. Press F5 in VSCode to build Ionide and start experimental instance
+2. Open Ionide-vscode-fsharp in VSCode.
+3. Run FSAC:
+    - or from sources in your favorite editor: open the `FsAutoComplete.sln`, start debugging with  `.NET Core mode http (debug)` configuration
+    - or run it with `--mode http --port 8088` arguments
+4. Press F5 in VSCode to build Ionide and start experimental instance
```

Comment:
You don't really need an experimental instance in this case. Just set the setting in any instance. This feature is enabling debugging FSAC without building Ionide at all, it may be worth to emphasise this in this guide.

Reply:
done